.env.example

```diff
@@ -0,0 +1,6 @@
+MAPBOX_TOKEN=
+DB_URL=
+SECRET=
+CLOUDINARY_KEY=
```

Comment:
This file is to tell anyone who clones your app, what env variables app needs to run